src/lineup/internal/LineUpPanelActions.ts

```diff
@@ -88,25 +88,31 @@ export default class LineUpPanelActions extends EventHandler {
 
     if (this.options.enableSidePanel === 'top') {
       this.node.classList.add('lu-side-panel-top');
-
     } else {
-      const sidePanel = new SidePanelTab(this.node, this.searchBoxProvider.createSearchBox(), ctx, doc);
+
+      const options = {
+        cssClass: 'fa fa-sliders',
+        title: 'Ranking Configuration',
+        width: '21em',
+        order: 0
+      };
+
+      const sidePanel = new SidePanelTab(this.node, this.searchBoxProvider.createSearchBox(), ctx, doc, options);
       this.panel = sidePanel.panel;
 
+      const listener = () => {
+        this.tabContainer.showTab(sidePanel);
+      };
       this.tabContainer = new PanelTabContainer(this.node);
-      this.tabContainer.addTab(sidePanel);
+      const lineupNavButton = new PanelNavButton(this.tabContainer.node, listener, options, true);
+      this.tabContainer.addTab(sidePanel, lineupNavButton);
```

Comment:
Do we need to create the PanelNavButton outside or could we create the button inside of `addTab`? Because the `PanelTab` (or here `SidePanelTab`) contains already the options.